plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -24,10 +26,13 @@ type InternetSpeed struct {
 	EnableFileDownload bool     `toml:"enable_file_download" deprecated:"1.25.0;use 'memory_saving_mode' instead"`
 	MemorySavingMode   bool     `toml:"memory_saving_mode"`
 	Cache              bool     `toml:"cache"`
+	Connections        int      `toml:"connections"`
+	MultiServerMode    bool     `toml:"multi_server_mode"`
```

Comment:
I think your last push removed these these settings from the example config and then when I ran make docs it removed them from the readme :) we need to document these settings.

Rather than a boolean, let's use a string `server_test_mode` with either `single` or `multi` as options.

Here is what you had at one point with a few updates, what do you think about this?

```
  ## Number of concurrent connections
  ## By default or set to zero, the number of CPU cores is used. Use this to
  ## reduce the impact on system performance or to increase the connections on
  ## faster connections to ensure the fastest speed.
  # connections = 0

  ## Sever test node
  ## By default, a single sever is used for testing. This may work for most,
  ## however, setting to "multi" will reach out to multiple servers in an
  ## attempt to get closer to ideal internet speeds.
  # server_test_mode = "single"
```

Reply:
Good, but I think change option name `server_test_mode` to `test_mode` is better.  What do you think?

Reply:
that works, thx